### PR TITLE
Added idempotent test case for site.yml

### PR DESCRIPTION
**Pull request by preethisn**

### CODE REVIEW

**Comment by preethisn** [comment]
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1595841756127/ -- Logs
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1595841756127/ceph_ansible_idempotent_0.log

**Comment by rakeshgm** [comment]
update the correct polarion id here and update the polarion also with correct status.

**Comment by preethisn** [comment]
Added polarion test case. 
CEPH-83573525

File: suites/nautilus/ansible/sanity_ceph_ansible.yaml [hunk 1]
```diff
@@ -43,6 +43,45 @@ tests:
       destroy-cluster: False
       abort-on-fail: true
 
+   - test:
+      name: ceph ansible_idempotent_site.yaml
+      polarion-id: CEPH-83573525
```
[the remark this answers is not in the document]
Comment (preethisn) [reply suites/nautilus/ansible/sanity_ceph_ansible.yaml:48]: Updated test steps in CEPH-83573525

**Pull request merged**